src/main/resources/db/migration/V1.10.0__NOTIF-163_rename_insights_bundle.sql

```diff
@@ -0,0 +1,18 @@
+--
+-- NOTIF-163
+-- Rename the 'insights' bundle to 'rhel'
+--
+
+-- Change bundle name and display-name
+UPDATE public.bundles
+SET name = 'rhel', display_name = 'Red Hat Enterprise Linux'
+WHERE
+  name = 'insights'
+;
+
+-- Update email susbscriptions
+UPDATE public.endpoint_email_subscriptions
+SET bundle = 'rhel'
+WHERE
+  bundle = 'insights'
```

Comment:
It is out of the scope of this PR, but this change shows of the things that surprised me while working on the database. Why are we using the bundle `name` in `endpoint_email_subscriptions` while we could have a foreign key based on `Bundle.id`?  It would make that kind of changes easier and improve data integrity as well.

Reply:
`app id` even, but yes, I thought the same, not sure why this is like this, I think things have deviated.